docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md

```diff
@@ -88,10 +91,10 @@ $[TIZEN_STUDIO_DIR]/tools/ide/bin$ ./tizen package -t wgt -s [CERTIFICATE_PROFIL
     </manifest>  
     ```
 
-    package id of .NET application is set like below :
-    __package="org.tizen.example.CrossTemplate1.Tizen"__
+    > [!NOTE]
+	> Ensure that the package ID of the Web application is set to **org.tizen.example.CrossTemplate1.Tizen**.
```

Comment:
Question. In this section creating .NET application is described. 
So shouldn't the id is for .NET application? Is that package ID for the Web application?

**Original :** package id of _.NET application_ is set like below :
**Updated :** Ensure that the package ID of the _Web application_ is set to **org.tizen.example.CrossTemplate1.Tizen**.

Reply:
Yes, you are right. Fixed it.

Reply:
Thanks for applying my comments @surbhig12 
However I'm still thinking of removed information from the original sentence.

Both line 65 and 95, in the description for the package id in the above sample code, "package=" is trimmed out. I'm not sure whether this trimming is good for the clarity.

Below are the changes in both lines.
line 65 :  
   __*package="TestHybrid"*__ => **TestHybrid**
line 95 : 
__package="org.tizen.example.CrossTemplate1.Tizen"__ => **org.tizen.example.CrossTemplate1.Tizen**

Would you please let me know your opinion?

Reply:
@mijin85cho,

I too agree, in the code, it should be package id.

@hakkyu-samsung,
Please let us know your views as well.

Reply:
@mijin85cho @surbhig12 
Sorry for the late response, I agree that the current word choices may confuse the readers on which element they should be looking for in the manifest files. I suggest we change the sentences number 2 and 4 of the numbered list to something like "Check the package id in the <manifest file name> file". Also, we should explicitly mention that the **package** attribute refers to the package id.

For example,

2. Check the application's package id in the **config.xml** file.

The **package** attribute of the **\<tizen:application\>** element refers to the application's package id, make sure that it is set to **TestHybrid**.

...

4. Check the application's package id in the **tizen-manifest.xml** file.

The **package** attribute of the **\<manifest\>** element refers to the application's package id, make sure that it is set to **org.tizen.example.CrossTemplate1.Tizen**.

Reply:
@hakkyu-samsung,

Thank you so much for the detailed explanation. I have modified accordingly.

Reply:
Hello, @hakkyu-samsung 

Would you please check the latest version and let us know your opinion?